src/parser/jobs/ast/modules/ArcanaTracking/ArcanaSuggestions.tsx

```diff
@@ -56,172 +59,160 @@ interface SleeveIcon {
 }
 
 interface CardLog extends CardState {
-	targetName?: string
-	targetJob?: string
+	targetName: string | null
+	targetJob: string | null
 }
 
-export default class ArcanaSuggestions extends Module {
+export default class ArcanaSuggestions extends Analyser {
 	static override handle = 'arcanaSuggestions'
 
 	static override title = t('ast.arcana-suggestions.title')`Arcana Logs`
 	static override displayOrder = DISPLAY_ORDER.ARCANA_TRACKING
 
 	@dependency private data!: Data
-	@dependency private combatants!: Combatants
 	@dependency private arcanaTracking!: ArcanaTracking
 	@dependency private timeline!: Timeline
+	@dependency private actors!: Actors
 
 	private cardLogs: CardLog[] = []
-	private partyComp: string[] = []
-
-	private PLAY: number[] = []
-
-	private SLEEVE_ICON: SleeveIcon = {}
+	private play: Array<Action['id']> = []
+	private sleeveIcon: SleeveIcon = {}
 
 	//div privates for output
-	private history: DIVWindow[] = []
-	private lastDIVFalloffTime: number = 0
-	private divCast: number = 0
+	private history: DivinationWindow[] = []
+	private divinationCast: number = 0
 	//end div privates
 
-	protected override init() {
-		PLAY.forEach(actionKey => {
-			this.PLAY.push(this.data.actions[actionKey].id)
-		})
+	override initialise() {
+		this.play = PLAY.map(actionKey => this.data.actions[actionKey].id)
 
-		this.SLEEVE_ICON = {
+		this.sleeveIcon = {
 			[SleeveType.NOTHING]: '',
 			[SleeveType.ONE_STACK]: this.data.statuses.SLEEVE_DRAW.icon,
 			[SleeveType.TWO_STACK]: 'https://xivapi.com/i/019000/019562.png',
 		}
 
 		//used for divination tracking
-		this.addEventHook('normalisedapplybuff', {to: 'player', abilityId: this.data.statuses.DIVINATION.id}, this.tryOpenWindow)
-		this.addEventHook('normalisedapplybuff', {by: 'player', abilityId: this.data.statuses.DIVINATION.id}, this.countDIVBuffs)
-		this.addEventHook('removebuff', {to: 'player', abilityId: this.data.statuses.DIVINATION.id}, this.tryCloseWindow)
-		this.addEventHook('cast', {by: 'player'}, this.onCast)
+		const divinationFilter = filter<Event>().type('statusApply').status(this.data.statuses.DIVINATION.id)
+		this.addEventHook(divinationFilter.target(this.parser.actor.id), this.tryOpenWindow)
```

Comment:
I am unsure how to implement this
If it helps, this specific section is going to be deleted for EW since divination won't be apart of ArcanaSuggestions